Source/Scene/Implicit3DTileContent.js

```diff
@@ -0,0 +1,573 @@
+import { TileOrientedBoundingBox } from "../Cesium.js";
```

Comment:
Reminder to fix the circular dependency between `Cesium3DTile` and `Implicit3DTileContent`.

We discussed using `.prototype` on the tile object to get around this.